modules/library/main/src/main/java/org/geotools/data/collection/ListFeatureCollection.java

```diff
@@ -185,14 +175,16 @@ public synchronized ReferencedEnvelope getBounds() {
      * @return 
      */
     protected ReferencedEnvelope calculateBounds() {
-        ReferencedEnvelope extent = new ReferencedEnvelope();
+        ReferencedEnvelope extent = ReferencedEnvelope.create( getSchema().getCoordinateReferenceSystem() );
         for( SimpleFeature feature : list ){
-            if( feature == null ) continue;
+            if( feature == null )
+                continue;
```

Comment:
we tend to use braces when placing a statement on a seperate line, check developers guide for sun-java-conventions + 100 cols.